.condarc-cuda10.0

```diff
@@ -2,5 +2,6 @@ channels:
   - rapidsai/label/cuda10.0
   - rapidsai-nightly/label/cuda10.0
   - nvidia/label/cuda10.0
+  - numba
```

Comment:
Numba 0.44 is released on conda-forge and defaults now so this is no longer needed
```suggestion
```